### ISSUE TITLE
Roundcube VestaCP error: Could not save new password.

### ISSUE THREAD

**Issue by jonathanf** [serghey-rodin/vesta#2186]
### Operating System (OS/VERSION):

CentOS 7

### VestaCP Version:

1.0.0.5

### Installed Software (what you got with the installer):

php-fpm, nginx, mysql

### Steps to Reproduce:

I can't change webmail password through roundcube with vesta driver 

![Captura de pantalla de 2021-11-22 22-16-29](https://user-images.githubusercontent.com/926635/142966444-bff7b31a-a8db-4c2f-8670-2f451463ec6e.png)

### Related Issues/Forum Threads:

Only issues from 2016

### Other Notes:

This happen in my 3 servers since update to 1.0.0 VestaCP

**Comment by jonathanf** [comment]
I tried many thinks but this doesn't work after update... Any idea?

**Comment by anton-reutov** [comment]
How we can reproduce, because i can't reproduce on our test server with centos 7

**Comment by jonathanf** [comment]
I don't know but, this was working on 0.9 with the recent update to 1.0.0.x this is failing in 2 servers, I try to do some stuff but still no working.

If you want I can provide access to the servers to investigate this issue and #2187 both happen on same servers

**Comment by anton-reutov** [comment]
yes, provide access to info@vestacp.com, in the title specify number ot this ticket 2186 and 2187

**Comment by jaapmarcus** [comment]
Unless VestaCP has updated their path / drive file it will not surprise me it is broken...

http://c.vestacp.com/0.9.8/debian/roundcube-driver.php

```
$send  = 'POST /reset/mail/ HTTP/1.1' . PHP_EOL
```

and 

https://github.com/serghey-rodin/vesta/blob/master/web/reset/mail/index.php

**Comment by jonathanf** [comment]
> yes, provide access to [info@vestacp.com](mailto:info@vestacp.com), in the title specify number ot this ticket 2186 and 2187

The credentials was send the past week.

**Comment by kdementiev-reinvently** [comment]
Any idea to fix this?

**Comment by jaapmarcus** [comment]
Undo the changes made in https://github.com/serghey-rodin/vesta/blob/master/web/reset/mail/index.php and revert to the old file. And it should work again.

**Comment by jonathanf** [comment]
> Undo the changes made in https://github.com/serghey-rodin/vesta/blob/master/web/reset/mail/index.php and revert to the old file. And it should work again.

It works, awesome @jaapmarcus I really appreciate this.

**Comment by jonathanf** [comment]
@anton-reutov the credential accesses was expired here and for #2187

**Comment by kdementiev-reinvently** [comment]
Guys, any idea how to fix this?

**Comment by jaapmarcus** [comment]
Revert the file https://github.com/serghey-rodin/vesta/blob/master/web/reset/mail/index.php to version -1 ... 

Or switch to a different control panel...

**Comment by pauldemichele** [comment]
Hi @jaapmarcus,
thanks for the update

I'm running the same setup as @kdementiev-reinvently has apart from the operating system (Ubuntu 18.04)
it has never worked for me and I do not have experience using different versions of vestacp 
what do you need from my side to get this thing sorted?

let me know, thanks in advance

**Comment by jaapmarcus** [comment]
See https://github.com/serghey-rodin/vesta/pull/2230

For the changes required....  Just modify the file and it should work. You can also the consider switching to HestiaCP or MyVesta.

**Comment by pauldemichele** [comment]
hey @jaapmarcus,
thanks for getting back to me

I will try to use the old driver and I'll let you know
I really appreciate all the hard work have been made to build this incredible product and I do not want to use any other platforms
